homeassistant/components/light/xiaomi_miio.py

```diff
@@ -257,9 +259,16 @@ def async_update(self):
 
             self._state = state.is_on
             self._brightness = ceil((255 / 100.0) * state.brightness)
+
+            if state.delay_off_countdown > 0:
+                delayed_turn_off = dt.utcnow() - timedelta(
```

Comment:
@balloob Is there a better way of using utcnow() here? The device provides a countdown in seconds.

Reply:
May be dt.utcnow().replace(microsecond=0).